### DISCUSSION TITLE
Meaning of “requires root” tag on plugins.

### DISCUSSION THREAD

**Discussion by Pragma808** [SteamDeckHomebrew/decky-loader#131] [Q&A] [answered]
What exactly does the “requires root” tag on plugins in the repo entail?

Do they have to be installed a special way? Are there extra steps required for them to work properly?

Ty, I really dig this project.

**Comment by WerWolv** [comment, marked as answer]
Hi.
It means that the plugins requested to have root access in its manifest.

It's basically just a warning for users to be careful when installing such plugins as root access means the plugin gets full access over the system. This is required for various plugins as they may want to interact with the Kernel or the system to control things (such as e.g PowerTools needs root access to control various performance settings) but many plugins can live without it as well.

**Reply by Pragma808** [reply]
Ah, exactly the scope of answer I was hoping for. Thank you!!
